qemu/tests/cfg/cpu_info_check.cfg

```diff
@@ -0,0 +1,10 @@
+- cpu_info_check:
+    type = cpu_info_check
+    start_vm = no
+    cpu_model_3_1_0 = Cascadelake-Server Icelake-Client Icelake-Server KnightsMill
+    cpu_model_4_2_0 = Cascadelake-Server-noTSX Icelake-Client-noTSX Icelake-Server-noTSX
+    cpu_model_4_2_0 += Cascadelake-Server-v1 Cascadelake-Server-v2 Cascadelake-Server-v3
+    cpu_model_4_2_0 += Icelake-Server-v1 Icelake-Server-v2 EPYC-v1 EPYC-v2
+    cpu_model_8_1_0 = Cooperlake Cooperlake-v1 Snowridge Snowridge-v1 Snowridge-v2
+    cpu_model_5_0_0 = EPYC-Rome EPYC-Rome-v1 EPYC-v3 Skylake-Server-noTSX-IBRS
+    cpu_model_5_0_0 += Skylake-Client-noTSX-IBRS
```

Comment:
@nanliu-r Would you please help double confirm above cpu models? Thanks.

Reply:
ok, will update at below comment for too long.

Reply:
@vivianQizhu Hi, I confirmed with yuhuang and we don't need to check the *-v cpu models now, so please update using the cpu_TYPES cpu models. Thanks a lot.
I'm ok with qemu3.1.0.

For qemu 4.2.0
cpu_model_8_2_0 = Cascadelake-Server-noTSX Icelake-Client-noTSX Icelake-Server-noTSX Skylake-Client-noTSX-IBRS Skylake-Server-noTSX-IBRS
cpu_model_8_2_1 = Cooperlake 
cpu_model_8_3_0 = EPYC-Rome